README.md

```diff
@@ -5,4 +5,146 @@
+func (p *AddFieldProcessor) Specification(context.Context) (sdk.Specification, error) {
+	return sdk.Specification{
+		Name:    "myAddFieldProcessor",
+		Summary: "Add a field to the record.",
+		Description: `This processor lets you configure a static field that will
+be added to the record into field .Payload.After. If the payload is not
+structured data, this processor will panic.`,
+		Version: "v1.0.0",
+		Author:  "John Doe",
+		Parameters: map[string]config.Parameter{
+			"field": {Type: config.ParameterTypeString, Description: "The name of the field to add"},
+			"name":  {Type: config.ParameterTypeString, Description: "The value of the field to add"},
+		},
+	}, nil
+}
+
+// Configure is called by Conduit to configure the processor. It receives a map
+// with the parameters defined in the Specification method.
+func (p *AddFieldProcessor) Configure(ctx context.Context, config map[string]string) error {
+	p.Field = config["field"]
+	p.Value = config["value"]
+	return nil
+}
+
+// Process is called by Conduit to process records. It receives a slice of
+// records and should return a slice of processed records.
+func (p *AddFieldProcessor) Process(ctx context.Context, records []opencdc.Record) []sdk.ProcessedRecord {
+	out := make([]sdk.ProcessedRecord, 0, len(records))
+	for _, record := range records {
+		record.Payload.After.(opencdc.StructuredData)[p.Field] = p.Value
+		out = append(out, sdk.SingleRecord(record))
+	}
+	return out
+}
+```
+
+You also need to add an entrypoint to your connector, since it will be run as a
+standalone WASM plugin:
+
+```go
+//go:build wasm
+
+package main
+
+import (
+	example "example.com/add-field-processor"
+	sdk "github.com/conduitio/conduit-processor-sdk"
+)
+
+func main() {
+	sdk.Run(&example.AddFieldProcessor{})
+}
+```
+
+With this you are set to build your processor. Note that we are building the
+processor as a WebAssembly module, so you need to set `GOARCH` and `GOOS`:
+
+```sh
+GOARCH=wasm GOOS=wasip1 go build -o add-field-processor.wasm cmd/main.go
+```
+
+The produced `add-field-processor.wasm` file can be used as a processor in
+Conduit. Copy it to the `processors` directory of your Conduit instance and
+configure it in the `processors` section of the `conduit.yaml` file.
+
+## FAQ
+
+### Why do I need to specify `GOARCH` and `GOOS`?
+
+Conduit uses [WebAssembly](https://webassembly.org) to run standalone processors.
+This means that you need to build your processor as a WebAssembly module. You can
+do this by setting the environment variables `GOARCH=wasm` and `GOOS=wasip1` when
+running `go build`. This will produce a WebAssembly module that can be used as a
+processor in Conduit.
+
+### How do I run a processor?
+
+You can run a processor by copying the WebAssembly module to the `processors`
+directory of your Conduit instance and configuring it in the `processors`
+section of the `conduit.yaml` file using the name the processor defines in its
```

Comment:
`conduit.yaml` is the Conduit configuration file. What processor configuration is available there?

Reply:
Oh, right, should have called it the pipeline configuration file 🤦

Reply:
I was like, what am I missing here?:D